.travis.yml

```diff
@@ -30,30 +32,28 @@ matrix:
     - php: '7.1'
       env:
         - COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.3'
+    - php: '8.0'
       env:
         - COMPOSER_FLAGS="--prefer-lowest"
 
     # Test each supported Symfony version with lowest supported PHP version
-    - php: '7.1'
+    - php: '5.5.9'
```

Comment:
This one must be kept on 7.1 as it's the minimal PHP version required by this bundle!